python/lsst/ts/tunablelaser/ascii.py

```diff
@@ -106,8 +140,8 @@ class AsciiRegister:
 
     Parameters
     ----------
-    port : `AsciiSerial`
-        A serial port that writes and reads from the TunableLaser converter
+    port : `TCPIPClient`
+        A TCP/IP stream that writes and reads from the TunableLaser converter
```

Comment:
In the doc string for the _file_ (at the very first line, not here -- but I can't add a comment up there) you say:
```
Notes
-----
The most important classes are the `AsciiSerial` class and the `AsciiRegister`
class as they contain the bulk of the functionality.
```
Please update the comment, since you renamed the `AsciiSerial` class.